plugins/uartdmx/UartDmxThread.cpp

```diff
@@ -122,17 +151,14 @@ void *UartDmxThread::Run() {
 void UartDmxThread::CheckTimeGranularity() {
   TimeStamp ts1, ts2;
   Clock clock;
-  /** If sleeping for 1ms takes longer than this, don't trust
-   * usleep for this session
-   */
-  const int threshold = 3;
 
   clock.CurrentMonotonicTime(&ts1);
   usleep(1000);
   clock.CurrentMonotonicTime(&ts2);
 
   TimeInterval interval = ts2 - ts1;
-  m_granularity = interval.InMilliSeconds() > threshold ? BAD : GOOD;
+  m_granularity = (interval.InMilliSeconds() > BAD_GRANULARITY_LIMIT) ?
+	  BAD : GOOD;
```

Comment:
/plugins/uartdmx/UartDmxThread.cpp:161:  Tab found; better to use spaces  [whitespace/tab] [1]
```suggestion
      BAD : GOOD;
```

Reply:
Done.